### PR TITLE
 Add VNICProfiles filtering according user permits

### PR DESCRIPTION
**Pull request by bond95**
This PR was built on https://github.com/oVirt/ovirt-web-ui/pull/813 . Only last commit is about vNIC Profiles.

Fixes: #738

### CODE REVIEW

**Comment by sjd78** [comment]
As a reminder - the changes to the UI components will also need to be made on top of #796.
(@bond95 , @gregsheremeta)

File: src/intl/messages.js [hunk 1]
```diff
@@ -378,6 +378,7 @@ export const messages: { [messageId: string]: MessageType } = {
   vnicProfileEmpty: '<Empty>',
   yearsShort: 'y',
   yes: 'Yes',
+  youHaveNoAllowedVnicProfiles: 'You have no permissions to use vNIC Profiles.',
```
[the remark this answers is not in the document]
Comment (bond95) [reply src/intl/messages.js]: Can be.

**Comment by bond95** [comment]
@sjd78 please, review.

**Comment by gregsheremeta** [comment]
ci build please

**Comment by gregsheremeta** [comment]
[ovirt-web-ui-1.5.0-0.20181024git768a09c.el7.noarch.rpm](https://jenkins.ovirt.org/job/oVirt_ovirt-web-ui_standard-check-pr/142/artifact/build-artifacts.el7.x86_64/ovirt-web-ui-1.5.0-0.20181024git768a09c.el7.noarch.rpm)

[ovirt-web-ui-1.5.0-0.20181024git768a09c.fc28.noarch.rpm](https://jenkins.ovirt.org/job/oVirt_ovirt-web-ui_standard-check-pr/142/artifact/build-artifacts.fc28.x86_64/ovirt-web-ui-1.5.0-0.20181024git768a09c.fc28.noarch.rpm)

**Comment by sjd78** [comment]
@bond95 - Now that #796 has been merged, this PR needs to be rebased and the permit filtering needs to be applied to the Nics card.

**Comment by bond95** [comment]
@gregsheremeta @sjd78 Rebased, also add filtering to dashboard view. Please, quick look.

**Pull request merged**